src/components/RenderInsideChart.jsx

```diff
@@ -11,18 +13,23 @@ class RenderInsideChart extends PureComponent {
         const at = this.props && this.props.at || 'holder';
 
         this.context.promise.then((context) => {
-            const stx = context.stx;
-            const elem = createElement('<div></div>');
-            const marker = stx.chart.panel[at].appendChild(elem);
-            this.marker = marker;
+            const nodeName = `${inChartPrefix}${at}`;
+            // reuse existing node when possible:
+            let elem = context.topNode.querySelector(`.${nodeName}`);
+            if (!elem) {
+                elem = createElement(`<div class="${nodeName}"></div>`);
+                context.stx.chart.panel[at].appendChild(elem);
+            }
+            this.container = elem;
+            this.forceUpdate(); // force render to be called after getting the container
```

Comment:
When repeatedly mounting and unmounting `<RenderInsideChart/>`, render is not called after children is `this.container` is set. We need to force render here.